README.md

```diff
@@ -1,79 +1,79 @@
-
-[Live Demo Link](https://livedemo.com)
-
-
-## Getting Started
-
-**I used HTML, CSS to implement the Mobile Version of the project, implemented the GitHub flow, and adopted best practices in the process of documenting the project.**
-
-
-## To request a local copy of my project, please follow these steps
-v2-desktop-version-with-animations
-Request this url: https://github.com/ogagaoghene/v1-desktop-version-with-animations in your web browser. Once opened navigate to the top left level of the project a green code download button will be visible on the righthand side. Select download Zip option from drop down menu. Once the download is complete you will be able to access my project locally.
-
-## Links (GitHub Page link for application deploy - v1-desktop-version-with-animations)
-https://ogagaoghene.github.io/v1-mobile-web-app/
-
-### Prerequisites
-
-### Setup
-
-### Install
-
-### Usage
-
-### Run tests
-
-### Deployment
-
-## Description about Accessibility Features
-
-- Added descriptive Page Title
-- Added alternative attributes for all images
-- Structure headings in order
-- Evaluate the color contrast
-- Resizing working fine
-- For interaction added tabindex to move around page content
-- Evaluate basic structure of the page
-
-## Authors
-👤 **Author1** 
-  Esi Ogagaoghene
-👤 **Author2** 
-  Ghazanfar Ali
-
-- GitHub: [@githubhandle](https://github.com/ogagaoghene)
-- Twitter: [@twitterhandle](https://twitter.com/@esi_ogaga)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/ogagaoghene-esi-7a478647)
-
-## 🤝 Contributing
-
-Contributions, issues, and feature requests are welcome!
-
-Feel free to check the [issues page](../../issues/).
-
-## Show your support
-
-Give a ⭐️ if you like this project!
-
-## Acknowledgments
-
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
-
-## 📝 License
-
-This project is [MIT](./MIT.md) licensed.
+# Project Name
+
+Portfolio: Mobile Version
+
+## Built With
+
+- html & CSS
+- Frameworks
+- vs code, github, gitbash
+- Figma hi-fi tool
+
+## Live Demo (if available)
+
+[Live Demo Link](https://livedemo.com)
+
+
+## Getting Started
+
+**I used HTML, CSS to implement the Mobile Version of the project, implemented the GitHub flow, and adopted best practices in the process of documenting the project.**
+
+
+## To request a local copy of my project, please follow these steps
+v2-desktop-version-with-animations
+Request this url: https://github.com/ogagaoghene/v1-desktop-version-with-animations in your web browser. Once opened navigate to the top left level of the project a green code download button will be visible on the righthand side. Select download Zip option from drop down menu. Once the download is complete you will be able to access my project locally.
+
+## Links (GitHub Page link for application deploy - v1-desktop-version-with-animations)
+https://ogagaoghene.github.io/v1-mobile-web-app/
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment

```

Comment:
- [ ] As requested by previous reviewer please remove these sections header to make your readme clear.